addons/sys_list/stringtable.xml

```diff
@@ -41,5 +41,54 @@
             <German>Zeigt Benachrichtigungen für Funk und Funkgeräte.</German>
             <Japanese>無線機と送信に関する通知を表示します。</Japanese>
         </Key>
+        <Key ID="STR_ACRE_sys_list_ACREUI_displayName">
+            <English>ACRE2 - User Interface</English>
```

Comment:
Those are not the strings I linked you: https://github.com/CBATeam/CBA_A3/blob/master/addons/ui/stringtable.xml#L40 (link goes to line 40).

```suggestion
            <English>ACRE2 UI</English>
```